packages/waas/src/auth.ts

```diff
@@ -363,6 +363,8 @@ export class SequenceWaaS {
   private async updateSessionStatus() {
     // if we are not signed in, then we don't need to check and update session status
     if ((await this.waas.isSignedIn()) === false) {
+      // just to be safe, remove any possible data related to previous session
+      await this.waas.completeSignOut()
```

Comment:
does `completeSignOut()` maybe any network requests or is it all local?

Reply:
all local